.github/workflows/build.yaml

```diff
@@ -77,8 +77,11 @@ jobs:
 
           cp ./scripts/livepeer-vmagent ./bin/livepeer-vmagent
 
+      - name: Uninstall rustup if preexists
+        run: rustup uninstall stable
```

Comment:
https://github.com/rust-lang/rustup/issues/1408 suggests uninstalling it first.

Reply:
Thanks for checking 👌 it's looking promising